### ISSUE TITLE
[Feature] 快速在多个AI中切换。

### ISSUE THREAD

**Issue by aaro-n** [ChatGPTNextWeb/ChatGPT-Next-Web#3883]
> 为了提高交流效率，我们设立了官方 QQ 群和 QQ 频道，如果你在使用或者搭建过程中遇到了任何问题，请先第一时间加群或者频道咨询解决，除非是可以稳定复现的 Bug 或者较为有创意的功能建议，否则请不要随意往 Issue 区发送低质无意义帖子。

> [点击加入官方群聊](https://github.com/Yidadaa/ChatGPT-Next-Web/discussions/1724)

**这个功能与现有的问题有关吗？**
无

**你想要什么功能或者有什么建议？**
[ChatGPT-Next-Web](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web)可以配置多个不同AI，但无法快捷切换不同AI，我希望ChatGPT-Next-Web可以在后台配置后 ，网页可以快速切换不同AI 。

**有没有可以参考的同类竞品？**
无

**其他信息**
如果要实现这个功能，需要修改ChatGPT-Next-Web后台和网页，我只是业余爱好者。
# 想法
ChatGPT-Next-Web 后台配置不同Ai访问地址和密钥，例如：
AI1：OPENAI
OPENAI地址，OPENAI访问密钥，OPENAI其他配置
AI2:AZURE_API
AZURE_API地址，AZURE_API密钥，AZURE_API其他配置
AI3:GOOGLE_API
GOOGLE_API地址，GOOGLE_API密钥 ，GOOGLE_API其他配置
AI4
......
AI5
.........
网页端需要添加选择不同Ai选项，并且以不同颜色标识，这样可以方便识别，例如：AI1红色，ai2青色，ai3橙色。

**Comment by AbnerChen028** [comment]
+1

**Comment by Tim-Saijun** [comment]
同，能同时调用多个AI就更好了

**Comment by AndyX-Net** [comment]
引用：https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/issues/4111

目前有一个解决方法，使用One-API作为中间层，可以匹配多个不同/相同的AI接入点：
https://github.com/songquanpeng/one-api

一旦配置好之后，你只需要在ChatGPT-Next-Web选择相应的模型即可。

**Comment by H0llyW00dzZ** [comment]
> 引用：#4111
> 
> 目前有一个解决方法，使用One-API作为中间层，可以匹配多个不同/相同的AI接入点： https://github.com/songquanpeng/one-api
> 
> 一旦配置好之后，你只需要在ChatGPT-Next-Web选择相应的模型即可。

Yes I recommended it, this is the most stable version I've used on my Kubernetes (k8s). It's a free setup with better performance, written in Go using the Gin package. Essentially, all you need is your own API key and host.
